apparmor.d/profiles-a-f/firewalld

```diff
@@ -62,20 +65,38 @@ profile firewalld @{exec_path} {
   /etc/firewalld/{,**} rw,
   /etc/iproute2/group r,
   /etc/iproute2/rt_realms r,
+  # Maybe change to as in kmod,lspci,...?
+  # /etc/modprobe.d/{,*.conf} r,
+  /etc/modprobe.d/ r,
+  /etc/modprobe.d/firewalld-sysctls.conf r,
 
   /var/lib/ebtables/lock rwk,
 
   /var/log/firewalld rw,
 
   @{run}/firewalld/{,*} rw,
+  @{run}/modprobe.d/ r, # Maybe change to as in kmod,lspci?
+                        # @{run}/modprobe.d/{,*.conf} r,
   @{run}/xtables.lock rwk,
 
+        @{PROC}/cmdline r,
         @{PROC}/sys/kernel/modprobe r,
         @{PROC}/sys/net/ipv{4,6}/ip_forward rw,
   owner @{PROC}/@{pid}/fd/ r,
   owner @{PROC}/@{pid}/mounts r,
   owner @{PROC}/@{pids}/net/ip_tables_names r,
 
+  @{sys}/module/compression r,
+  # Maybe change to as in systemd-modules-load?
```

Comment:
No, here, we can limit module reading to the network one needed by firewalld. So keep it this way.
Also, `@{sys}/module/compression r,` and `@{PROC}/cmdline r,` are part of `app/kmod`. And `@{sys}` rules must be moved before `@{PROC}` rules.

Reply:
I can find the `@{PROC}/cmdline r,` but not `@{sys}/module/compression r,` in `abstractions/app/kmod` or `abstractions/consoles`. Are you sure it's there?

Reply:
My bad, you are right.